### DISCUSSION TITLE
value assignment inadmissible

### DISCUSSION THREAD

**Discussion by lin-ycv** [visose/Robots#189] [Q&A] [answered]
Used the "UseController" option for my tool, but when it tries to read the load data it throws an error
If I remove that line everything still works fine, but I'd like to know why it's throwing the error
![20221013_112913](https://user-images.githubusercontent.com/60565737/197007624-16bf4a39-166f-483b-bc8f-065c400fb3dd.jpg)

**Comment by visose** [comment, marked as answer]
Find `LOAD_DATA[5]` in the file `R1/System/$config.dat` of the controller. Is it assigned to a valid value?
